tests/tools/plugins/test_cppapi.cc

```diff
@@ -115,9 +115,20 @@ f()
   ALWAYS_ASSERT(c2.asTSCont() != nullptr)
   ALWAYS_ASSERT(c2.mutex() == m)
 
-  ALWAYS_ASSERT(!c)
-  ALWAYS_ASSERT(c.asTSCont() == nullptr)
-  ALWAYS_ASSERT(c.mutex() == nullptr)
+  // NOLINTNEXTLINE
```

Comment:
I think clang-analyzer wants pre-processor used to hide/ignore stuff from it:

eg

#if defined(__clang_analyzer__)

#endif

https://clang-analyzer.llvm.org/faq.html#exclude_code

Reply:
Thank you Randal! Yes, I was trying to treat this as a clang-tidy issue via NOLINT comments but it was instead a clang-format one. The test passes now that I've implemented your feedback.